src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -185,7 +185,7 @@
          [quo/button
           {:on-press
            (if config/community-accounts-selection-enabled?
-             #(rf/dispatch [:open-modal :community-account-selection community])
+             #(rf/dispatch [:open-modal :community-account-selection {:community-id id}])
              #(rf/dispatch [:open-modal :community-requests-to-join community]))
```

Comment:
Should we pass the argument as line above?
instead of `comunity` we use `{:community-id id}`?